docs/recipes/CodeSplitting.md

```diff
@@ -0,0 +1,138 @@
+Another approach is to create a 'Reducer Manager' object, which keeps track of all the registered reducers and exposes a `reduce()` function. Consider the following example:
+
+```javascript
+export function createReducerManager(initialReducers) {
+    // Create an object which maps keys to reducers
+    const reducers = { ...initialReducers };
+
+    // Create the initial combinedReducer
+    let combinedReducer = combineReducers(reducers);
+
+    // An array which is used to delete state keys when reducers are removed
+    const keysToRemove = [];
+
+    return {
+        getReducerMap: () => reducers,
+
+        // The root reducer function exposed by this object
+        // This will be passed to the store
+        reduce: (state, action) => {
+            // If any reducers have been removed, clean up their state first
+            if (keysToRemove.length > 0) {
+                state = { ...state as any };
+                for (let key of keysToRemove) {
+                    delete state[key];
+                }
+                keysToRemove = [];
+            }
+
+            // Delegate to the combined reducer
+            return combinedReducer(state, action);
+        },
+
+        // Adds a new reducer with the specified key
+        add: (key, reducer) => {
+            if (!key || reducers[key]) {
+                return;
+            }
+
+            // Add the reducer to the reducer mapping
+            reducers[key] = reducer;
+
+            // Generate a new combined reducer
+            combinedReducer = combineReducers(reducers);
+        },
+
+        // Removes a reducer with the specified key
+        remove: (key: string) => {
+            if (!key || !reducers[key]) {
+                return;
+            }
+
+            // Remove it from the reducer mapping
+            delete reducers[key];
+
+            // Add the key to the list of keys to clean up
+            keysToRemove.push(key);
+
+            // Generate a new combined reducer
+            combinedReducer = combineReducers(rm);
+        }
+    };
+}
+
+const staticReducers = {
+    users: usersReducer,
+    posts: postsReducer
+};
+
+export function configureStore(initialState) {
+    const reducerManager = createReducerManager(staticReducers);
+
+    // Create a store with the root reducer function being the one exposed by the manager.
+    const store = createStore(reducerManager.reduce, initialState);
+
+    // Optional: Put the reducer manager on the store so it is easily accessible
+    store.reducerManager = reducerManager;
+}
+```
+
+To add a new reducer, one can now call `store.reducerManager.add("asyncState", asyncReducer)`.
+
+To remove a reducer, one can now call `store.reducerManager.remove("asyncState")`
+
+## Libraries and Frameworks
+There are a few good libraries out there that can help you add the above functionality automatically:
+ * [`redux-dynamic-reducer`](https://github.com/ioof-holdings/redux-dynamic-reducer)
```

Comment:
Hey, I'm the author/maintainer of `redux-dynamic-reducer` and while I'm stoked you want to include my library in these docs, it's unfortunately out of active development and [will be deprecated soon](https://github.com/ioof-holdings/redux-dynamic-reducer/issues/16).  

Luckily, we do have a new library called [`redux-dynostore`](https://github.com/ioof-holdings/redux-dynostore) which supersedes it, adding more configurability and dynamic sagas support and you still get the `addReducers` function on the store.